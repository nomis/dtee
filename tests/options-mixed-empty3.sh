#!/bin/bash
set -vx
. "$(dirname "$0")"/util/common.sh

# Empty command stops all following options parsing
run_test "--debug-options" "" "-o" "filename1" "-e" "filename2"
RET=$?

echo RET $RET
if [ $RET -eq 0 ]; then
	exit 0
fi
exit 1
