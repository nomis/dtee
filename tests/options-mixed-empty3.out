-=
-=-o
-=filename1
-=-e
-=filename2
debug-options=1
